test/ffmpeg-vaapi/decode/10bit/hevc.py

```diff
@@ -9,6 +9,8 @@
 from ..decoder import DecoderTest
 
 spec = load_test_spec("hevc", "decode", "10bit")
+spec_422 = load_test_spec("hevc", "decode", "10bit", "422p")
+spec_444 = load_test_spec("hevc", "decode", "10bit", "444p")
```

Comment:
Maybe it would be better not to introduce a new spec key.  Rather, we could just filter the original/existing spec based on format value during tests `@slash.parametrize` call.  For example:

```
@slash.parametrize(("case"), sorted([k for k,v in spec.items() if v["format"] in ["P010"]]))
```

Reply:
have change it